src/ci/resources/tes_application.conf.ctmpl

```diff
@@ -0,0 +1,76 @@
+include required(classpath("application.conf"))
+include "build_application.inc.conf"
+
+filesystems {
+    blob {
+        class = "cromwell.filesystems.blob.BlobPathBuilderFactory"
+        global {
+            class = "cromwell.filesystems.blob.BlobFileSystemManager"
+            config {
+                workspace-manager: {
```

Comment:
In some cases we want to test with WSM, and in some cases we want to test TES without it. We could edit this file, but I'm wondering whether it would be valuable to have two different config templates and two different TES run configurations, one with Terra+WSM support and one without.

Reply:
Good call! I split the configs into `terra_tes_application.conf.ctmpl` and `tes_application.conf` so we can use a rendered config for WSM stuff and a non-rendered, WSM-free config otherwise.